src/borg/compress.pyx

```diff
@@ -433,6 +435,44 @@ class Auto(CompressorBase):
         raise NotImplementedError
 
 
+class ObfuscateSize(CompressorBase):
+    """
+    Meta-Compressor that obfuscates the compressed data size.
+    """
+
+    ID = b'\x04\x00'
+    name = 'obfuscate'
+
+    def __init__(self, compressor=None):
+        super().__init__()
+        self.compressor = compressor
+        self.factor = 0.1  # approx. relative amount of obfuscation data to add
```

Comment:
as this is the maximum addtl. amount (== 10%), I guess the repo would grow about 5%.